website/docs/r/schema_template_contract_service_graph.html.markdown

```diff
@@ -3,98 +3,63 @@ layout: "mso"
-* `node_relationship.provider_connector_redirect_policy` - (Optional) redirection policy for provider connector at site level.
-* `node_relationship.consumer_connector_redirect_policy_tenant` - (Optional) tenant for redirection policy for consumer connector at site level. It is required to set redirection policy for consumer connector.
-* `node_relationship.consumer_connector_redirect_policy` - (Optional) redirection policy for consumer connector at site level.
-
-* `node_relationship.provider_subnet_ips` - (Optional) subnet ips which will be associated with provider connector at site level. It should be in CIDR format.
-* `node_relationship.consumer_subnet_ips` - (Optional) subnet ips which will be associated with consumer connector at site level. It should be in CIDR format.
-
-
+* `schema_id` - (Required) The schema ID of the Contract Service Graph.
+* `template_name` - (Required) The template name of the Contract Service Graph.
+* `contract_name` - (Required) The contract name of the Contract Service Graph.
+* `service_graph_name` - (Required) The name of the Service Graph.
+* `service_graph_schema_id` - (Optional) The schema ID of the Service Graph. The `schema_id` will be used if not provided.
+* `service_graph_template_name` - (Optional) The template name of the Service Graph. The `template_name` will be used if not provided.
+* `node_relationship` - (Required) The Contract Service Graph Node relationship information.
+  * `provider_connector_bd_name` - (Required) The name of the BD that has to be connected to a Provider Connector.
+  * `provider_connector_bd_schema_id` - (Optional) The schema ID of the BD that has to be connected to a Provider Connector. The `schema_id` will be used if not provided.
+  * `provider_connector_bd_template_name` - (Optional) The template name of the BD that has to be connected to a Provider Connector. The `template_name` will be used if not provided.
+  * `consumer_connector_bd_name` - (Required) The name of the BD that has to be connected to a Consumer Connector.
+  * `consumer_connector_bd_schema_id` - (Optional) The schema ID of the BD that has to be connected to a Consumer Connector. The `schema_id` will be used if not provided.
+  * `consumer_connector_bd_template_name` - (Optional) The template name of the BD that has to be connected to a Consumer Connector. The `template_name` will be used if not provided.
 
 ## Attribute Reference ##
 
 No attributes are exported.
 
 ## Importing ##
 
-An existing MSO Schema Template Contract service graph can be [imported][docs-import] into this resource via its Id/path, via the following command: [docs-import]: <https://www.terraform.io/docs/import/index.html>
+An existing MSO Schema Template Contract Service Graph can be [imported][docs-import] into this resource via its Id/path, via the following command: [docs-import]: <https://www.terraform.io/docs/import/index.html>
```

Comment:
An existing MSO Schema Template Contract Service Graph can be [imported][docs-import] into this resource using its Id/path, via the following command: [docs-import]: <https://www.terraform.io/docs/import/index.html>